onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h

```diff
@@ -25,7 +25,6 @@ namespace Dml
     class ReadbackHeap;
     class ExecutionContext;
     class BucketizedBufferAllocator;
-    class CPUAllocator;
```

Comment:
It seems okay from this *local view*, but repeating some conversations in chat, you note there are some checks in the DML EP looking for this memory type, and that the proper fix may be to change ORT extensions to check mem_type rather than specific allocator name.